mockintosh/handlers.py

```diff
@@ -660,10 +665,20 @@ def __init__(self):
         self.path = None
         self.headers = {}
         self.queryString = {}
-        self.body = None
```

Comment:
You have removed `body` attribute while code still uses it. Looks not right

Reply:
Fixed with https://github.com/up9inc/mockintosh/commit/55a7dfb21b8abb046fa10d675f90cf976c91e3f8